webpack/redux/actions/RedHatRepositories/helpers.js

```diff
@@ -45,9 +45,9 @@ const recommendedRepositoriesMisc = [
   'satellite-capsule-6.16-for-rhel-8-x86_64-rpms',
   'satellite-maintenance-6.16-for-rhel-8-x86_64-rpms',
   'satellite-utils-6.16-for-rhel-8-x86_64-rpms',
```

Comment:
```suggestion
```
These can be removed since we don't need Satellite 6.17 to be recommending Satellite 6.16 repositories.